src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+const calendar = document.querySelector('.calendar');
+
+const daysInMonth = 31;
```

Comment:
You have hardcoded the number of days in a month. This might be fine for now, but it's not a scalable solution. What if you need to generate a calendar for February or April? Consider passing the number of days as an argument to your function, or even better, calculate it based on the month and year.